### ISSUE TITLE
Save program memory space by storing strings in once

### ISSUE THREAD

**Issue by z3t0** [z3t0/Arduino-IRremote#261]
Kudos: @pcoughlin @ElectricRCAircraftGuy
Related: #258 

This is an issue about decreasing the footprint of this library by storing all strings into the flash memory at the beginning and only once.

**Comment by bengtmartensson** [comment]
Smartass mode: You mean __constant__ strings of course...

I made a quick scan, and except for commented-out stuff and debug stuff (that is not compiled unless DEBUG is defined (or true?), I could not find any constant strings. (Except for `ir_Pronto.cpp` but that is another story...) So it appears that there is nothing to do. If I am wrong, please correct me.

**Comment by z3t0** [comment]
I think this issue was before we had DEBUG so it is no longer relevant.
